jujugui/static/gui/src/app/components/usso-login-link/_usso-login-link.scss

```diff
@@ -0,0 +1,26 @@
+.usso-login {
+  position: relative;
+
+  &__notification {
+    background: #fff;
+    box-shadow: 0 1px 4.85px 0.15px rgba(0,0,0,0.2);
+    box-sizing: border-box;
+    color: #666;
+    font-size: 14px;
+    line-height: normal;
+    margin: 10px auto 0;
+    padding: 12px 25px;
+    position: absolute;
+    z-index: 3;
```

Comment:
Replace the hardcoded z-index with this:

~~~~
z-index: index($z-indexed-elements, usso-login-link);
~~~~

Then add 'usso-login-link' to the list here: https://github.com/juju/juju-gui/blob/develop/jujugui/static/gui/src/app/assets/css/base.scss#L38